### ISSUE TITLE
SDLD od SDLS

### ISSUE THREAD

**Issue by Kishore4c9** [kirillv/cpp-inverse-kinematics-library#1]
In "TODO" it was mentioned "ADD SDLD Solver"
Is it SDLD or SDLS. 
What is the necessity of SDLS and CCD Solvers?
What is their role in here

**Comment by kirillv** [comment]
You are right - I meant SDLS. The idea was yo implement all well known algos for IK problem solving.

**Comment by nixwang** [comment]
How to add CCD Solver? thank you
